2023-round-2/tiwavaldese/nx_dev_test_output.txt

```diff
@@ -0,0 +1,1083 @@
+================================================== warnings summary ===================================================
+networkx\networkx\utils\backends.py:576
+  C:\Users\00\networkx\networkx\utils\backends.py:576: DeprecationWarning:
+
+  random_tree is deprecated and will be removed in NX v3.4
+  Use random_labeled_tree instead.
+    return self.orig_func(*args, **kwargs)
+
+networkx\networkx\readwrite\tests\test_gml.py:557
+  C:\Users\00\networkx\networkx\readwrite\tests\test_gml.py:557: DeprecationWarning: invalid octal escape sequence '\420'
+    "graph [edge [ source u'u\4200' target u'u\4200' ] "
+
+networkx\networkx\readwrite\tests\test_gml.py:558
+  C:\Users\00\networkx\networkx\readwrite\tests\test_gml.py:558: DeprecationWarning: invalid octal escape sequence '\420'
+    + "node [ id u'u\4200' label b ] ]"
+
+networkx/networkx/linalg/tests/test_algebraic_connectivity.py::TestSpectralOrdering::test_cycle[lobpcg-False-expected_order0]
+  C:\Users\00\networkx\networkx\linalg\algebraicconnectivity.py:302: UserWarning: Exited at iteration 10 with accuracies
+  [0.02743716]
+  not reaching the requested tolerance 1e-08.
+  Use iteration 11 instead with accuracy
+  0.02743715868521578.
+
+    sigma, X = sp.sparse.linalg.lobpcg(
+
+networkx/networkx/linalg/tests/test_algebraic_connectivity.py::TestSpectralOrdering::test_cycle[lobpcg-False-expected_order0]
+  C:\Users\00\networkx\networkx\linalg\algebraicconnectivity.py:302: UserWarning: Exited postprocessing with accuracies
+  [0.02743716]
+  not reaching the requested tolerance 1e-08.
+    sigma, X = sp.sparse.linalg.lobpcg(
+
+networkx/networkx/linalg/tests/test_algebraic_connectivity.py::TestSpectralOrdering::test_cycle[lobpcg-True-expected_order1]
+  C:\Users\00\networkx\networkx\linalg\algebraicconnectivity.py:302: UserWarning: Exited at iteration 10 with accuracies
+  [0.00056623]
+  not reaching the requested tolerance 1e-08.
+  Use iteration 11 instead with accuracy
+  0.0005662307712154139.
+
+    sigma, X = sp.sparse.linalg.lobpcg(
+
+networkx/networkx/linalg/tests/test_algebraic_connectivity.py::TestSpectralOrdering::test_cycle[lobpcg-True-expected_order1]
+  C:\Users\00\networkx\networkx\linalg\algebraicconnectivity.py:302: UserWarning: Exited postprocessing with accuracies
+  [0.00056623]
+  not reaching the requested tolerance 1e-08.
+    sigma, X = sp.sparse.linalg.lobpcg(
+
+-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
+=============================================== short test summary info ===============================================
+FAILED networkx/networkx/drawing/tests/test_pydot.py::test_pydot_issue_258 - FileNotFoundError: [WinError 2] "neato" not found in path.
+FAILED networkx/networkx/drawing/tests/test_pydot.py::test_pydot_numerical_name - FileNotFoundError: [WinError 2] "dot" not found in path.
+==================== 2 failed, 5092 passed, 56 skipped, 3 xfailed, 7 warnings in 282.04s (0:04:42) ====================
```

Comment:
Thanks for this! The tests are failing. Could you look into this? You can also try removing pydot so pytest skips the tests.

Reply:
Thank you @MridulS it pass now without any fail.

please does it means i have done my first contribution already?

Reply:
Can you please update the test output file with the new output then? Thanks!

Reply:
yeah i did already and push